### ISSUE TITLE
Another run of the Kin-RFCs

### ISSUE THREAD

**Issue by v1thesource** [kinecosystem/rewards-engine#11]
Participation in the Kin-RFCs back in Oct/Nov was likely dampened due to a combination of grim market conditions and fewer eyes on the project (some of whom did not share our enthusiasm for the Kin project). Many did not understand the true scope of the project back then, assuming it was just confined to the Kik Messenger. I think the community has now had time to learn about the vision of the Kin project, and can therefore now provide more meaningful contributions.

Over the past few months, Kin's audience and number of token holders has greatly increased, with more active community members who are willing to contribute ideas to the project. I believe within the next few weeks, as excitement around the Stellar migration arrives and community momentum and morale is on the increase, we should have another push for contributions here. Utilise the new full-time community managers to encourage participation and put out a quick Medium post. 

Thoughts?

**Comment by ryana** [comment]
I think the relation to market timing is a pretty astute observation. For what it's worth, I've been helping out the originprotocol.com team and we're having some success with this kind of open collaboration, but we had to rely on Slack to do it. I think the lack of live discussion on Github makes it hard to move things forward meaningfully, as people are stuck wondering if anyone is paying attention (whereas in Slack it's easier to know if people are seeing your messages but ignoring you 😂).

I know the Kin ICO slack was a bloodbath, but I think the team should revisit building a public dev slack and pair it with discussion on Github.

I also just looked up and remembered I had this tab open: https://www.youtube.com/watch?v=_DPftmg7zR8. I would love to participate in a Google hangout like that on the KRE (or at the very least be able to watch others!)

**Comment by odednoam** [comment]
Challenge accepted :)

We'll try to find the best way to discuss the changes to the KRE with the goal of releasing a new revision within a few weeks. I don't think slack/telegram/hangouts discussions are too useful becuase I find them hard to follow in retrospect. I prefer that we start with something more structured such as github issues (like this one) and revisions. I opened issue #12 to discuss the change I want to introduce, of a "KRE for start-ups" plan. With smaller changes to the document, I'll open separate issues and relate them to commits to the document.

If this doesn't work we can try telegram or hangouts.

Thanks guys
Oded
